lms/djangoapps/discussion/rest_api/tests/test_api.py

```diff
@@ -2684,6 +2696,7 @@ def test_basic(self, parent_id):
             "children": [],
             "editable_fields": ["abuse_flagged", "raw_body", "voted"],
             "child_count": 0,
+            "can_delete": True,
```

Comment:
Just last question, why "can_delete" is not part of the "editable_fields"?

Reply:
One approach I had in mind was to just make "deleted" an editable field, however "delete" is a pure operation, it needs a call using the delete verb, and doesn't correspond to a field. 

In all other cases, if a field is marked as editable, that is the field in the body that you can change, it would no longer be the case with "can_delete".